### ISSUE TITLE
config.ini issue (Parsing issue happen again)

### ISSUE THREAD

**Issue by KorewaLidesu** [neon-nyan/CollapseLauncher#12]
No log avail, here is config.ini
I have talking about this issue before on DIscord (Korewa_Li#0001)

```[General]
channel=1
cps=
game_version=5.8.0
sub_channel=1
sdk_version=

older/Hi3SEA
game_start_name=BH3.exe
is_first_exit=False
exit_type=2

=2```

**Comment by KorewaLidesu** [comment]
Also, about Install/Update function, can you try optional for requirement of compressed game file?

**Comment by neon-nyan** [comment]
Hi There,
Sorry for my late reply for this as I'm kind of busy on my RL job this week.
Yes, the bug on INI Parser is still happening and I'm planning to move to use [**this project**](https://github.com/rickyah/ini-parser) or using build-in Win32 INI Parser via **Interop**

Also, for the second question, Sorry if I'm not quite understand but did you mean the feature that allows you to use existing game installation?

**Comment by neon-nyan** [comment]
Uhm, that's weird. The launcher should still using the existing game installation and shouldn't redownload the entire thing again.
Could you check if the config.ini file inside of the App Data folder is also getting broken? (the one you can open by going to "App Settings" > "Open App Data Folder")

**Comment by KorewaLidesu** [comment]
```[launcher]
channel=1
cps=
game_version=5.8.0
sub_channel=1
sdk_version=
game_install_path=D:\GAME\CollapseFolder\Hi3SEA
game_start_name=BH3.exe
is_first_exit=False
exit_type=2```

I use the similar one that you suggest on Discord.

**Comment by neon-nyan** [comment]
Uhm, config file doesn't seem to have an issue.
Can you try to locate the installation location to the existing installation once you're getting asked by the launcher to install the game?
You can click on Install Game > Then click "Locate Location" and choose the location where the BH3.exe should be located

**Comment by KorewaLidesu** [comment]
> Uhm, config file doesn't seem to have an issue. Can you try to locate the installation location to the existing installation once you're getting asked by the launcher to install the game? You can click on Install Game > Then click "Locate Location" and choose the location where the BH3.exe should be located

Yes, have done it before. Still give required an update.
P/s: Use Delta patch actually skip the requirement for update, but after config.ini issue occur, it is required too update again.

~~sorry if it is hard for you to understand, me just poor english :teri_cry:~~

**Comment by neon-nyan** [comment]
That's surely weird.
Anyway, Hope I can investigate about this further and anyway, what version and which release channel (stable/preview) you're using right now?

**Comment by KorewaLidesu** [comment]
I'm using stable branch (latest version)

Vào 12:39, Th 7, 9 thg 7, 2022 Kemal Setya Adhi ***@***.***>
đã viết:

**Comment by neon-nyan** [comment]
Okay, thanks for the info. I will leave this issue open and hope I can fix this in the next version
![](https://cdn.discordapp.com/emojis/672927172633165825.png)

**Comment by neon-nyan** [comment]
Hi there,
This INI Parsing issue has been addressed in 1.0.33.1 release.

If you don't have any question for this, I think we can close this issue.
Also, thank you for reporting this. Hope this launcher could improve over time.
